util/test.js

```diff
@@ -0,0 +1,11 @@
+import { IS_TESTNET } from '@/constant';
+
+const getTestAttribute = name => (
+  IS_TESTNET ?
+    {
+      'lc-test': name,
+    } :
+    null
+);
```

Comment:
Wrap this into a function so that you can inject prefix, then directly use it for selector without #id 
```javascript
const getTestAttribute = prefix => (testId) => (
  IS_TESTNET ? ({ 
    'lc-test': `${prefix}-${testId}`,
  }) : (
    null
  )
);
```